lib/input-unmarshaller.js

```diff
@@ -30,8 +30,8 @@ module.exports = class InputUnmarshaller extends Transform {
         const inputIndex = dataSignal.getArgindex();
         const rawPayload = dataSignal.getPayload();
         const contentType = dataSignal.getContenttype();
-        const acceptedContentType = determineContentTypes(contentType).accept;
```

Comment:
This seemed wrong. Why were we looking at "accept"?

Reply:
That's what https://github.com/jshttp/negotiator returns as the accepted media type.